src/components/FocusTrap/TOP_TAB_SCREENS.ts

```diff
@@ -1,5 +1,10 @@
+import type {TupleToUnion} from 'type-fest';
 import CONST from '@src/CONST';
 
-const TOP_TAB_SCREENS: string[] = [CONST.TAB.NEW_CHAT, CONST.TAB.NEW_ROOM, CONST.TAB_REQUEST.DISTANCE, CONST.TAB_REQUEST.MANUAL, CONST.TAB_REQUEST.SCAN];
+const TOP_TAB_SCREENS = [CONST.TAB.NEW_CHAT, CONST.TAB.NEW_ROOM, CONST.TAB_REQUEST.DISTANCE, CONST.TAB_REQUEST.MANUAL, CONST.TAB_REQUEST.SCAN] as const;
```

Comment:
NAB: We could consider transferring these CONSTs to SCREENs because as a matter of fact they are screens; It would also match rest of screens listed in `getAdaptedStateFromPath.ts`, but I leave it for C+ to decide 👍